azext_iot/_params.py

```diff
@@ -495,6 +496,41 @@ def load_arguments(self, _):
             "Account and Contributor role for the IoT Hub.",
         )
 
+    with self.argument_context("iot hub device-identity generate-mqtt-credentials") as context:
+        context.argument(
+            "connection_string",
+            options_list=["--connection-string", "--cs"],
+            help="Target connection string. This bypasses the IoT Hub registry and generates the user credentials directly "
+                 "from the supplied symmetric key without further validation. All other command parameters aside from "
+                 "password-creation-time and password-expiry-time will be ignored. Supported connection string types: Device, "
+                 "Module."
+        )
+        context.argument(
+            "password_expiry_time",
+            options_list=["--password-expiry-time", "--pet"],
+            help="Integer representing time in seconds representing the password expiry time.",
```

Comment:
Would be good to say the default expiry of 1 hour if omitted (as its currently implemented).